tests/foreman/ui/test_host.py

```diff
@@ -161,11 +164,10 @@ def setUpClass(cls):
         # Search if Libvirt compute-resource already exists
         # If so, just update its relevant fields otherwise,
         # Create new compute-resource with 'libvirt' provider.
-        resource_url = u'qemu+tcp://{0}:16509/system'.format(
-            settings.server.hostname
+        resource_url = u'qemu+ssh://root@{0}/system'.format(
+            settings.compute_resources.libvirt_hostname
```

Comment:
So yeah the way the code it is right now, I belive it will throw a runtime exception if the required property is not set.  I am asking to skip the test if the property is not set.